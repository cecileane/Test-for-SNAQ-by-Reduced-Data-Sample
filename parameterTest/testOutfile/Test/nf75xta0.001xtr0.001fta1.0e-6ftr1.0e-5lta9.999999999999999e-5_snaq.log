optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-6,
                       xtolAbs=0.001, xtolRel=0.001.
 max number of failed proposals = 75, liktolAbs = 9.999999999999999e-5.
rootname for files: nf75xta0.001xtr0.001fta1.0e-6ftr1.0e-5lta9.999999999999999e-5_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 9355
seed: 9355 for run 1
Sat Oct  1 17:25:42 2016

 BEGIN SNaQ for run 1, seed 9355 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 166 (failures: 75) with -loglik=155.49626 and ht_min=[0.33535,0.12573,0.55437,0.32983]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 44		 44		 3	 |	 2		 2		 --
mvtarget	 51		 51		 2	 |	 1		 1		 --
chdir		 17		 17		 1	 |	 4		 4		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 53		 35		 2	 |	 5		 5		 --
Total		 166		 148		 9	 |	 12		 12		 8
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.7

 FINISHED SNaQ for run 1, -loglik of best 155.49625734556747
(((4,#H7:::0.37302474023989235):2.1558607135955814,(1,(2)#H7:::0.6269752597601077):2.1558607135955814):0.12573304013730785,(5,6):0.3353467776906152,3);
---------------------
seed: 44980 for run 2
Sat Oct  1 17:25:53 2016

 BEGIN SNaQ for run 2, seed 44980 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 85 (failures: 75) with -loglik=126.74251 and ht_min=[0.2856,0.48942,10.0,0.51916,1.51213]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 23		 23		 0	 |	 0		 0		 --
mvtarget	 25		 25		 0	 |	 0		 0		 --
chdir		 5		 5		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 30		 26		 0	 |	 2		 2		 --
Total		 85		 80		 2	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 126.74250981975379
(3,((2,(1)#H9:::0.7143969488695707):1.512133452923744,4):0.48941900464472826,(5,(6,#H9:::0.2856030511304292):10.0):0.5191607531783365);
---------------------

Sat Oct  1 17:25:59 2016
MaxNet is (3,((2,(1)#H9:::0.7143969488695707):1.512133452923744,4):0.48941900464472826,(5,(6,#H9:::0.2856030511304292):10.0):0.5191607531783365); 
with -loglik 126.74250981975379
