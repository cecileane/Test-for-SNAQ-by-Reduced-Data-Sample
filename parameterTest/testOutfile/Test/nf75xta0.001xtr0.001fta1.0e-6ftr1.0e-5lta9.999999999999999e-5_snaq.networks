(3,((2,(1)#H9:::0.7143969488695707):1.512133452923744,4):0.48941900464472826,(5,(6,#H9:::0.2856030511304292):10.0):0.5191607531783365);, with -loglik 126.74250981975379 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,((2,(1,#H9:::0.3249716002036761):2.421082483388155):0.6596387094577273,4):0.46300112337029203,(5,(6)#H9:::0.6750283997963239):2.011297453206535);, with -loglik 128.4875130349389
(3,(4)#H9:::0.6190898545911001,(5,(6,(1,(2,#H9:::0.38091014540889995):2.9740971309977366):1.167922671416438):0.2196669578861788):0.04100932195092845);, with -loglik 172.4488856018328
(3,(#H9:::0.4255687079914773,4):1.8768611914526023,(5,(6,(1,(2)#H9:::0.5744312920085227):4.543677447138041):0.15793138896712833):0.023306806053564923);, with -loglik 192.03261646762232
(5,(6,(1,(2,((3)#H9:::0.6039548349454116,4):0.07655335953862621):0.09739270521134637):1.0016362103528382):7.388553278589985,#H9:::0.3960451650545885);, with -loglik 215.17319476299338
(3,((2,(1,(#H9:::0.4807909361913817,6):5.492445640204222):0.045018218055835696):0.2159291283162021,4):1.0791731482658349,(5)#H9:::0.5192090638086183);, with -loglik 231.3898426350714
