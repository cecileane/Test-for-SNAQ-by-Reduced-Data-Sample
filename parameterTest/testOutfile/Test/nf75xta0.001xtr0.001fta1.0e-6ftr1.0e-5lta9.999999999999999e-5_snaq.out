(3,((2,(1)#H9:::0.7143969488695707):1.512133452923744,4):0.48941900464472826,(5,(6,#H9:::0.2856030511304292):10.0):0.5191607531783365); -Ploglik = 126.74250981975379
 Dendroscope: (3,((2,(1)#H9):1.512133452923744,4):0.48941900464472826,(5,(6,#H9):10.0):0.5191607531783365);
 Elapsed time: 16.669563281 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (3,((2,(1)#H9:::0.7143969488695707):1.512133452923744,4):0.48941900464472826,(5,(6,#H9:::0.2856030511304292):10.0):0.5191607531783365);, with -loglik 126.74250981975379
 (((4,#H7:::0.37302474023989235):2.1558607135955814,(1,(2)#H7:::0.6269752597601077):2.1558607135955814):0.12573304013730785,(5,6):0.3353467776906152,3);, with -loglik 155.49625734556747
-------
